### ISSUE TITLE
cant run mlat-server --help

### ISSUE THREAD

**Issue by GregoryGHarding** [mutability/mlat-server#6]
i have installed all the pre-reqs, but when i attempt to run mlat-server --help
i get: "mlat-server: command not found"

any guidance would be greatly appreciated

**Comment by mutability** [comment]
Probably "." isn't in your PATH, try ./mlat-server

**Comment by GregoryGHarding** [comment]
Tried ./mlat-server --help

Results in..

Traceback (most recent call last):
  File "./mlat-server", line 21, in <module>
    import mlat.server.main
ImportError: No module named 'mlat.server'

**Comment by mutability** [comment]
It works for me, so there's something about your python install that you'll have to work out.

**Comment by GregoryGHarding** [comment]
ok, i knew there wasnt much support for the server, so thanks for trying anyway

if its any help, the client software runs fine

**Comment by mutability** [comment]
Ah, if you have the client software installed systemwide, that interferes with the server packages. Try uninstalling the client, or run the server under virtualenv.

**Comment by GregoryGHarding** [comment]
im running a VM now with only server and prerequisites installed. does this give any hints as to whats going on? 

![untitled](https://cloud.githubusercontent.com/assets/12652473/11644935/afc34368-9d2a-11e5-8cb0-4928ecd8f804.png)

**Comment by mutability** [comment]
Isn't the error message selfexplanatory?

**Comment by GregoryGHarding** [comment]
yes, i got that, i posted wrong screen, the error im getting says numpy module not found. but it is installed, ive checked import numpy

**Comment by GregoryGHarding** [comment]
is it possible its defaulting installs to python 2.7 and not 3?

**Comment by mutability** [comment]
The shebang line explictly invokes python3.4, so nope, it's not that. When you check that numpy is importable, are you running python3?

**Comment by GregoryGHarding** [comment]
thats the thing, i run python and import numpy, it goes fine, then run python3 import numpy fails

**Comment by mutability** [comment]
Show me exactly what you're running and what you get back.

**Comment by GregoryGHarding** [comment]
ive gone back and installed each prerequisite using python3 instead of just python command and i can use mlat-server now. sorry to be a pain i now you dont like providing alot of support. is it possible you can throw up an assembled simple example mlat-server arguments

**Comment by mutability** [comment]
```
#!/bin/sh
D=`dirname $0`
MLAT_CPU_PROFILE=1 $D/mlat-server \
  --client-listen 40147:40147 \
  --basestation-connect localhost:31003 \
  --filtered-basestation-connect localhost:31004 \
  --write-csv $D/positions.csv \
  --work-dir $D \
  --check-leaks \
  >>$D/server.log 2>&1 &
echo "$!" >$D/server.pid
```

**Comment by GregoryGHarding** [comment]
trying to activate the tools/sync and coverage pages

i have apache installed and have synlinked tools to www root. am i missing anything because the page is discriptive text and dosnt refresh to show sync status, same for coverage

sorry for asking alot of questions, im trying to learn/understand this software the best i can

**Comment by mutability** [comment]
Coverage maps requires that you generate the tiles separately e.g. from cronjob.
The sync page requires that you make sync.json (generated by the server) available in the same dir - e.g. symlink it.

**Comment by big23** [comment]
How to install it on RPi ?

I have still error like Gregory :(
(Please update AGPL_SERVER_CODE_URL )

Pliiiz help

**Comment by jairforce1** [comment]
Hello.
 I can not start the server with the same error.
”RuntimeError: Please update AGPL_SERVER_CODE_URL in mlat/server/config.py”
How is this update done?
![default](https://user-images.githubusercontent.com/25606111/26982993-3021a2d8-4d75-11e7-8346-2dfac299991d.png)

**Comment by eivindotcom** [comment]
Do as the RuntimeError says.

Go into
`mlat/server/`
use editor to open config.py
`nano config.py`
Find the line 

"#AGPL_SERVER_CODE_URL = "https://github.com/mutability/mlat-server"

update the link / remove hashtag

**Comment by jairforce1** [comment]
Thank you.
The update issue has been resolved.
However, other errors can not be resolved yet.
I installed the prerequisite module but it does not work.
The import of modules seems to go wrong.

**Comment by jairforce1** [comment]
Installation of pykalman is completed
However, Pearlman's Ehler comes out.

![default](https://user-images.githubusercontent.com/25606111/27083521-cc648a3c-5083-11e7-87de-4fa2d2aa28c6.png)

**Comment by GKVELT** [comment]
Hello I also have a problem with mlat.server.main. I can't import this module and I don't know where to find it!
Can anyone help me? I tried a lot of version of python in Windows and macOS. 
Thanks!
